### ISSUE TITLE
'open foo.rkt' shows up twice in menu 

### ISSUE THREAD

**Issue by mfelleisen** [racket/drracket#138]
I will send a screenshot in reply to witness this bug.

**Comment by mfelleisen** [comment]
[the promised screenshot as a link](http://homedirs.ccs.neu.edu/matthias/Tmp/for-issue-138-gitgub-drracket.png)

**Comment by rfindler** [comment]
What is hiding underneath the menu? (The obvious guesses aren't making it show up for me.)

**Comment by mfelleisen** [comment]
I was in bar.rkt: 

#lang racket
(require (prefix-in abc: "foo.rkt"))
(+ abc:xyz (abc:xyz 10))

Here is foo.rkt: 

#lang racket
(provide xyz)
(define xyz 10)

**Comment by rfindler** [comment]
Sadly, that doesn't make it show up for me.

Are these files in your homedirectory?

**Comment by rfindler** [comment]
![screen shot 2017-08-17 at 9 58 37 am](https://user-images.githubusercontent.com/399024/29418718-bcbc76c6-8332-11e7-8d04-0ff6aa322de5.png)

**Comment by mfelleisen** [comment]
No, they are in a directory labeled ‘Bugs’ inside of Images inside of 2htdp

**Comment by mfelleisen** [comment]
What does this mean? That you fixed it or that you can’t reconstruct the bug? Is it possibly something that was fixed recently I haven’t updated in a week.

**Comment by rfindler** [comment]
I could not reconstruct it. But @dfeltey tells me he knows what is going on, so we're all good for now.

**Comment by dfeltey** [comment]
I think the problem is that check-syntax isn't doing some filtering of menu
items when it should be. I started working on a change that would prevent
this a while ago, I'll make time to get back to it  this week.
